docproc/src/main/java/com/yahoo/docproc/DocprocExecutor.java

```diff
@@ -176,4 +181,15 @@ private void logProgress(Processing processing, DocumentProcessor.Progress progr
         }
         return progress;
     }
+
+    private Function<String, Metric.Context> cachedContexts(String chainDimension) {
+        Map<String, Metric.Context> contextCache = new ConcurrentHashMap<>();
+        return documentType -> contextCache.computeIfAbsent(documentType, type -> {
+            Map<String, String> dimensions = new HashMap<>(2);
+            dimensions.put("chain", chainDimension);
+            dimensions.put("documentType", type);
```

Comment:
In all backend metrics this dimension is named "documenttype". We should use the same her I guess.

Reply:
Geir is correct. The dimension name must be exactly "documenttype", or it cannot be used in combination with other metrics. The option would likely be to replicated "documenttype" into "documentType" for all other metrics having this already, if we conclude "documentType" is the better name.

Reply:
Thanhks.